domain/secret/state/state.go

```diff
@@ -2900,6 +2875,43 @@ AND    (subject_type_id = $secretAccessorType.unit_type_id AND subject_id IN ($u
 	return revisionResult, nil
 }
 
+// GetSecretRevisionID returns the revision UUID for the specified secret URI and version,
+// or an error satisfying [secreterrors.SecretRevisionNotFound] if the revision is not found.
+func (s *State) GetSecretRevisionID(ctx context.Context, uri *coresecrets.URI, version int) (uuid.UUID, error) {
```

Comment:
s/version/revisionNum

Or even just "revision"
We never used the word "version" anywhere else